src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final ColorSupplier supplierOfColors = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        return new Ball(supplierOfColors.getRandomColor(), new Random().nextInt(100));

```

Comment:
same for Random. And [All magic numbers in your code should be constants](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#all-magic-numbers-in-your-code-should-be-constants)